### DISCUSSION TITLE
Is there any built-in utility to transliterate on an index?

### DISCUSSION THREAD

**Discussion by georgiosd** [ravendb/ravendb#18918] [Q&A] [answered]
Hey guys

This bug took me about 1 hour of staring at the screen going "it should be working" 🤣 

Basically on a map-reduce index, I was grouping by a property that on one document was A (English) and on another it was Α (Greek).

If you can't see the difference, that's why I lost an hour of my day 👎 

In my web project I use [this package](https://github.com/anyascii/anyascii) to get over this - is there a built-in way to handle this on an index?

Thank you

**Comment by ayende** [comment, marked as answer]
You can just use `anyascii` see:

https://ravendb.net/docs/article-page/6.0/csharp/indexes/additional-assemblies
